cns/restserver/internalapi.go

```diff
@@ -191,6 +196,13 @@ func (service *HTTPRestService) ReconcileNCState(ncRequest *cns.CreateNetworkCon
 		}
 	}
 
+	ctx := context.Background()
```

Comment:
need a cancellable context, but this function is short lived

Reply:
Curious, why is this being moved from main? It seems like in this scope (`ReconcileNCState`) the monitor would get "started" multiple times rather than just once.